CONTRIBUTING.md

```diff
@@ -11,3 +11,5 @@ Contributions in the form of pull requests, issue reports, and feature requests
 - [ ] Send a pull request (PR) and work with us until it gets merged and published. Contributions may need some modifications, so a few rounds of review and fixing may be necessary.
 
 For quick merging, the contribution should be short, and concentrated on a single feature or topic. The larger the contribution is, the longer it would take to review it and merge it.
+
+Please follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) rule when writing commit messages.
```

Comment:
Need to add more info, like that users would have to install the hook explicitly via:
```
pre-commit install --hook-type commit-msg --hook-type pre-push
```